### ISSUE TITLE
Saving configuration

### ISSUE THREAD

**Issue by vorlonempire** [ottokiksmaler/gear360_modding#4]
Hi. I know this is not the best place to ask a question like that but maybe it wont hurt :)
How can i save my configuration once i made some changes by st cap capdtm setusr X 0x000etc
After reboot setting are not saved and i cant find any option to save those settings i changed
Thanks for the answer!

**Comment by ottokiksmaler** [comment]
On NX500 I use the sequence in batch file executed upon boot, but I have not tried that with gear 360. It's not the best solution either way since st commands take a while to execute.

**Comment by vorlonempire** [comment]
Yeah, thats the solution i came up with also, but it would be cool if there would be a way to save these changes permanently
